src/auth/access-group-provider/access-group-from-graphql-api-call.service.ts

```diff
@@ -0,0 +1,46 @@
+import { AccessGroupService as AccessGroupService } from "./access-group.service";
+import { Injectable } from "@nestjs/common";
+import fetch from "node-fetch";
```

Comment:
@nitrosx I think instead of introducing new library like `node-fetch` we can use `@nestjs/axios`. Here is an example: https://docs.nestjs.com/techniques/http-module

Reply:
@Martin: I agree with you.
Let's leave it like that and I will create a new ticket.